src/Sylius/Bundle/ApiBundle/Command/Cart/RemoveItemFromCart.php

```diff
@@ -17,22 +17,19 @@
 
 class RemoveItemFromCart implements OrderTokenValueAwareInterface
 {
-    public function __construct(public ?string $orderTokenValue, public string $itemId)
-    {
+    public function __construct(
+        protected string $itemId,
+        protected string $orderTokenValue,
+    ) {
     }
 
-    public static function removeFromData(string $tokenValue, string $orderItemId): self
+    public function getItemId(): ?string
```

Comment:
```suggestion
    public function getItemId(): string
```
type to be discussed, but atm it's not nullable